Assets/MRTK/SDK/Profiles/DefaultMixedRealitySpatialAwarenessMeshObserverProfile.asset

```diff
@@ -22,6 +22,6 @@ MonoBehaviour:
   levelOfDetail: 0
   trianglesPerCubicMeter: 0
   recalculateNormals: 1
-  displayOption: 1
+  displayOption: 2
```

Comment:
@davidkline-ms Will this have any unexpected side effects in other uses of this profile?

Reply:
This should be applied to only MRTKExamplesHub. I'll revert this change.